lib/omnibus/git_cache.rb

```diff
@@ -129,16 +129,19 @@ def restore
 
       create_cache_path
 
-      cmd = shellout(%Q(git --git-dir=#{cache_path} --work-tree=#{install_dir} tag -l "#{tag}"))
-
       restore_me = false
-      cmd.stdout.each_line do |line|
-        restore_me = true if tag == line.chomp
+      begin
+        cmd = git_cmd(%Q(tag -l "#{tag}"))
+
+        cmd.stdout.each_line do |line|
+          restore_me = true if tag == line.chomp
+        end
+      rescue CommandFailed
```

Comment:
You're right.  I was being lazy about investigating why this call was using shellout instead of shellout! but it seems to be a historical artifact. I switched it to not catch any exceptions.